### PR TITLE
Describe video acceleration and other make movie parameters

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related with https://github.com/omichel/webots/pull/6174.

Add the decsription of the make movie dialog and the various option including the new recording in slow motion feature.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: guide/the-user-interface.md [hunk 1]
```diff
@@ -102,9 +102,20 @@ to save the current view as a PNG or JPG image.
...
 create MPEG movies (Linux and macOS) or AVI movies (Windows). Once the movie
 recording is started, this item is changed in `Stop Movie...`. During the
-recording, it is possible to the change the running mode and pause the
+recording, it is possible to change the running mode and pause the
 simulation. However, frames are only captured during Webots steps and not when
 the simulation is paused.
+Other than choosing the resolution and the compression quality of the movie, it is also possible to record the simulation in accelerated mode or in slow motion by setting the `Video acceleration` value in the pop-up dialog.
+If the video acceleration value is below 1, the recorded movie will run slower than the simulation.
+The maximum slow down, i.e. the minimum acceleration value, is defined by the basic time step of the simulation because it is not possible to record at a higher frame rate than the simulation update rate.
+In order to increase the maximum slow down please consider reducing the simulation basic time step.
```
Comment (omichel) [inline guide/the-user-interface.md]: I would replace "please consider reducing" by "you should reduce".

File: guide/the-user-interface.md [hunk 2]
```diff
@@ -102,9 +102,20 @@ to save the current view as a PNG or JPG image.
...
+Then, by checking the video caption option the current acceleration value can be displayed on the top right corner of the 3D scene.
```
Comment (omichel) [inline guide/the-user-interface.md]: I would rephrase it this way: "To display the acceleration value in the top right corner of the movie, you should check the video caption option." or "Checking the video caption option will display the acceleration value in the top right corner of the movie."

**Comment by stefaniapedrazzi** [comment]
:+1:
